okio/src/jvmTest/kotlin/okio/DeflaterSinkTest.kt

```diff
@@ -129,6 +128,29 @@ class DeflaterSinkTest {
     mockSink.assertLogContains("close()")
   }
 
+  /**
+   * This test deflates a single segment of without compression because that's
+   * the easiest way to force close() to emit a large amount of data to the
+   * underlying sink.
```

Comment:
```suggestion
   * This test deflates a single segment without compression because that's
   * the easiest way to force close() and emit a large amount of data to the
   * underlying sink.
```

Reply:
I think the comment is garbage, copy and pasted.  I'll update to a more succinct and correct one.